.github/CODEOWNERS

```diff
@@ -1114,3 +1114,9 @@
 
 #PRLabel: %CodeGen %Client
 /eng/scripts/typespec/                                             @JoshLove-msft @m-nash @jorgerangel-msft @jsquire
+
+# PRLabel: %Liftr-Lite
+/sdk/liftrpinecone/Azure.ResourceManager.*/                        @aggarwalsw @kanikataneja @jawt @mrigankchugh
+/sdk/liftrweightsandbiases/Azure.ResourceManager.*/                @aggarwalsw @kanikataneja @jawt @mrigankchugh
```

Comment:
This is not a valid format.   You can only have a single path under the `PRLabel` metadata in a block.   You'll need to break each path up into an individual block with associated label metadata.